features/security/README.md

```diff
@@ -29,9 +29,64 @@ This component has its own test suite.
 
 ### Strong typing
 
-Strong typing is ensured by [our strong typing functional test suite](strong_typing.md) and [the unit tests of the `AbstractItemNormalizer`
+Strong typing is ensured by [our "strong typing" functional test suite](strong_typing.feature) and [the unit tests of the `AbstractItemNormalizer`
```

Comment:
I don't get it. Links seem to work (except those terminating with `.md` instead of `.feature`as you pointed out): https://github.com/dunglas/core/blob/33035a3859f6617ffabddb66a09d301565451a47/features/security/README.md